docs/source/ubuntu-opens-install.rst

```diff
@@ -360,8 +360,8 @@ On a compute node, perform the following steps:
     configuration is outside the scope of this install document.
 
     If you *are* configuring a full BGP mesh you'll need to handle the BGP
-    configuration appropriately. You should consult the relevant
-    documentation for your chosen BGP stack.
+    configuration appropriately. To configure BIRD as a route reflector,
+    follow the instructions in :doc:`bird-rr-config`.
 
```

Comment:
This is a tricky one. The short argument is that I wanted the RR instructions to work even if the user didn't do this stuff, so I used instructions that will always work. If you think that's unnecessary, I can rewrite.